### ISSUE TITLE
Feature Suggestion: Automatically open steam minimized to tray when switching

### ISSUE THREAD

**Issue by drkui** [TcNobo/TcNo-Acc-Switcher#251]
Hi,

Like the title suggests, could we have an option where Steam automatically gets minimized to tray when the Steam account is switched?

**Comment by Niggolo** [comment]
Yeah definitely that is was it missing, I have to close it every time
